deploy/helm/sumologic/values.yaml

```diff
@@ -1587,6 +1587,9 @@ otelcol:
       pullPolicy: IfNotPresent
   # To enable collecting all logs, set to false
   logLevelFilter: true
+  # Metrics from Collector
+  metrics_enabled: true
```

Comment:
This should be camelCase like the rest of the file.

Reply:
Or actually I have better idea, it should be like we have for Fluent Bit:
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/6a0ededa19f44ddd9ea36c96ac0a8be42f2a69b7/deploy/helm/sumologic/values.yaml#L729-L730

Reply:
I've created another PR for that one: https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1037